src/view/com/util/List.tsx

```diff
@@ -69,6 +83,9 @@ function ListImpl<ItemT>(
     },
     onEndDrag(e, ctx) {
       onEndDragFromContext?.(e, ctx)
+      if (isAndroid) {
+        runOnJS(updateActiveViewAsync)
```

Comment:
doesn't seem like you're actually calling the function here, shouldn't it be `runOnJS(updateActiveViewAsync)()`

Reply:
same below